docker/musicbrainz-server-base/Dockerfile

```diff
@@ -0,0 +1,149 @@
+        libdata-page-perl \
+        libdata-uuid-mt-perl \
+        libdate-calc-perl \
+        libdatetime-format-duration-perl \
+        libdatetime-format-iso8601-perl \
+        libdatetime-format-natural-perl \
+        libdatetime-format-pg-perl \
+        libdatetime-format-w3cdtf-perl \
+        libdatetime-timezone-perl \
+        libdb-dev \
+        libdbd-pg-perl \
+        libdbi-perl \
+        libdbix-connector-perl \
+        libdigest-hmac-perl \
+        libdigest-md5-file-perl \
+        libdigest-sha-perl \
+        libemail-address-perl \
+        libemail-mime-creator-perl \
+        libemail-mime-perl \
+        libemail-sender-perl \
+        libemail-valid-perl \
+        libencode-detect-perl \
+        libexception-class-perl \
+        libexpat1-dev \
+        libgnupg-perl \
+        libhtml-formhandler-perl \
+        libhtml-tiny-perl \
+        libhtml-treebuilder-xpath-perl \
+        libhttp-date-perl \
+        libicu-dev \
+        libintl-perl \
+        libio-all-perl \
+        libjson-perl \
+        libjson-xs-perl \
+        liblist-allutils-perl \
+        liblist-moreutils-perl \
+        liblist-utilsby-perl \
+        liblog-dispatch-perl \
+        libmath-random-secure-perl \
+        libmethod-signatures-simple-perl \
+        libmodule-pluggable-perl \
+        libmoose-perl \
+        libmoosex-abc-perl \
+        libmoosex-clone-perl \
+        libmoosex-getopt-perl \
+        libmoosex-methodattributes-perl \
+        libmoosex-role-parameterized-perl \
+        libmoosex-singleton-perl \
+        libmoosex-types-perl \
+        libmoosex-types-structured-perl \
+        libmoosex-types-uri-perl \
+        libmro-compat-perl \
+        libnet-amazon-awssign-perl \
+        libnet-coverartarchive-perl \
+        libpackage-stash-perl \
+        libpq-dev \
+        libreadonly-perl \
+        libredis-perl \
+        librest-utils-perl \
+        libset-scalar-perl \
+        libsoap-lite-perl \
+        libstarlet-perl \
+        libstatistics-basic-perl \
+        libstring-camelcase-perl \
+        libstring-shellquote-perl \
+        libstring-tt-perl \
+        libtemplate-plugin-class-perl \
+        libtemplate-plugin-javascript-perl \
+        libtemplate-plugin-json-escape-perl \
+        libtext-diff3-perl \
+        libtext-markdown-perl \
+        libtext-trim-perl \
+        libtext-unaccent-perl \
+        libtext-wikiformat-perl \
+        libthrowable-perl \
+        libtime-duration-perl \
+        libtrycatch-perl \
+        libunicode-icu-collator-perl \
+        liburi-perl \
+        libxml-generator-perl \
+        libxml-parser-perl \
+        libxml-rss-parser-lite-perl \
+        libxml-semanticdiff-perl \
+        libxml-simple-perl \
+        libxml-xpath-perl \
+        libxml2-dev \
+        nodejs \
+        # needed by ./admin/psql. trusty only has 9.3 in the official repos,
+        # but it works fine against a 9.5 server.
+        postgresql-client-9.3 && \
```

Comment:
I used the PG repo in the postgres Dockerfile. But got lazy about copying it here since the 9.3 client worked. This actually might be good to factor out into an m4 file.